fileremover/fileremover.go

```diff
@@ -0,0 +1,24 @@
+package fileremover
+
+import "os"
+
+// FileRemover ...
+type FileRemover interface {
```

Comment:
I think this functionality is generic enough so that we could export it to `go-utils`; probably not now but it might worth adding a `// TODO` here with a link to a JIRA ticket.

Reply:
opened a PR: https://github.com/bitrise-io/go-utils/pull/142